src/pages/CheckoutDetails.jsx

```
@@ -0,0 +1,107 @@
+import React, { useEffect, useState } from "react";
+import $axios from "../utils/axios";
+import { useSpinner } from "../hook/SpinnerContext";
+import { toast, ToastContainer } from "react-toastify";
+
+const Table = ({ rows, title }) => (
```

ESLint

[error] 'rows' is missing in props validation

(react/prop-types)